matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/sync/FilterService.kt

```diff
@@ -22,9 +22,9 @@ interface FilterService {
         NoFilter,
 
         /**
-         * Filter for Riot, will include only known event type
+         * Filter for Element, will include only known event type
          */
-        RiotFilter
+        ElementFilter
```

Comment:
should we not remove this from SDK?  and all stuff related to Element filter ofc

Reply:
Yes, probably.